homeassistant/components/fastdotcom/sensor.py

```diff
@@ -1,56 +1,133 @@
-"""Support for Fast.com internet speed testing sensor."""
+"""Sensor platform for the Fast.com integration."""
 
 from __future__ import annotations
 
+from collections.abc import Callable
+from dataclasses import dataclass
+
 from homeassistant.components.sensor import (
     SensorDeviceClass,
     SensorEntity,
+    SensorEntityDescription,
     SensorStateClass,
 )
-from homeassistant.const import UnitOfDataRate
+from homeassistant.const import UnitOfDataRate, UnitOfTime
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
-from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from .const import DOMAIN
+from .const import DEFAULT_NAME, DOMAIN
 from .coordinator import FastdotcomConfigEntry, FastdotcomDataUpdateCoordinator
 
 
+@dataclass(frozen=True)
+class FastdotcomSensorEntityDescription(SensorEntityDescription):
+    """Describes Fast.com sensor entities."""
+
+    value: Callable[[float], float] = lambda val: round(val, 2)
```

Comment:
can we set the suggested display precision instead of rounding? or do the values have too much decimals?